src/app/screens/LeaveScreen/TabScreen.tsx

```diff
@@ -100,6 +100,7 @@ function TabScreen({route}: Props) {
           isPendingRoute={route === 'pending'}
           startDate={dateRange.startDate}
           endDate={dateRange.endDate}
+          setDateRange={setDateRange}
```

Comment:
instead of passing `setDateRange` state function, create `resetDateRange` function and then pass into components.

Reply:
Okay